### ISSUE TITLE
Close issues solved by new accessibility improvements

### ISSUE THREAD

**Issue by nickcolley** [alphagov/accessible-autocomplete#356]
## What
Once we have merged https://github.com/alphagov/accessible-autocomplete/pull/355 we want to go through the issue tracker and close any related issues.

## Why
- ensure that we know what issues remain
- help us write the release notes.

## Depends on

- Refinements to address accessibility issues #355

## Done when

- [x] Issues fixed by the recent work are [added to the release milestone](https://github.com/alphagov/accessible-autocomplete/milestone/3)
- [ ] Issues are closed

**Comment by nickcolley** [comment]
When I did the initial review from the audit I was aware of these issues that seemed to be fixed by this new work:

- [ ] JAWS and NVDA form fields dialogue do not show autocomplete clearly [#344 (accessible-autocomplete)](https://github.com/alphagov/accessible-autocomplete/issues/344)
- [ ] JAWS 17/18 doesn't announce the number of results available [#235 (accessible-autocomplete)](https://github.com/alphagov/accessible-autocomplete/issues/235)
- [ ] Overlayed options cannot be swiped to on VoiceOver iOS [#345 (accessible-autocomplete)](https://github.com/alphagov/accessible-autocomplete/issues/345)
- [ ] NVDA unable to use 'escape' to exit the autocomplete [#346 (accessible-autocomplete)](https://github.com/alphagov/accessible-autocomplete/issues/346)
